lib/src/publication/remote.dart

```diff
@@ -235,11 +235,11 @@ class RemoteTrackPublication<T extends RemoteTrack>
   }
 
   @Deprecated('use setVideoQuality() instead')
```

Comment:
if this is deprecated and the PR is part of v2, we can probably remove it here (unless this is removed in the deprecation PR already, then ignore me :) )

Reply:
yes, he will be deleted in another PR. I changed it here just to not break the dart analyzer.